samples/f77/Makefile.in

```diff
@@ -1,10 +1,11 @@
 include @make_Cantera_dot_mak@
 
-F77=@F77@
+FC=@F77@
```

Comment:
Excellent question, and I share your thinking. I was merely trying to revert to a 'known' working state. From a quick google search it appears that `FC` needs to be set appropriately, and otherwise defaults to `f77` (which is the error I am getting). Using `FC` below works - it appears that `F77` is another variable that is set automatically.

Reply:
Apart from GitHub actions (which are still running), I confirmed that `FC` works on Linux and macOS (Apple silicon).